development.env

```diff
@@ -0,0 +1,8 @@
+RABBIT_HOST=localhost
```

Comment:
Is this loaded in anywhere or just for us to know what the defaults are?

Reply:
It's not used automatically. Could put it in the README instead of it's own file if you think that's clearer?

Reply:
Yeah that looks better 👍

Reply:
I woud like to have it default to dev values locally but I could see an immediate way of doing this nicely and I thought the priority should be having it require proper env config in prod settings, might revisit this later to see if we can make the dev config behaviour a bit nicer.